cnxauthoring/storage/postgresql.py

```diff
@@ -366,3 +366,7 @@ def search(self, limits, type_=Document, submitter_id=None):
             for item in res:
                 yield self._reassemble_model_from_document_entry(**item)
         raise StopIteration
+
+    def restart(self):
+        """Restart the interface"""
+        self.conn = psycopg2.connect(self.conn.dsn)
```

Comment:
From what I remember `self.conn.dsn` does not store the database password. (https://github.com/Connexions/cnx-authoring/pull/192#discussion_r23728193)  It was ok last time because it was in a test.  Since this is actually in a running service, we have to make sure it works when the database requires a password.